typings/pypandoc/__init__.pyi

```diff
@@ -0,0 +1,5 @@
+import pathlib
+
+def convert_file(
+    source_file: str, to: str, format: str | None, outputfile: str | pathlib.Path | None
+) -> str: ...
```

Comment:
Can you explain what's happening here? I see it being used to convert to .docx, but I don't see any return statement in this function.